src/main/java/org/dspace/ctask/replicate/TransmitAIP.java

```diff
@@ -44,11 +46,13 @@
 {
     // Group where all AIPs will be stored
     private String storeGroupName;
+    private String skipList;
 
     @Override
     public void init(Curator curator, String taskId) throws IOException {
         super.init(curator, taskId);
         storeGroupName = configurationService.getProperty("replicate.group.aip.name");
+        skipList = configurationService.getProperty("replicate.transmitaip.skiplist");
```

Comment:
Could this not use the new `getArrayProperty(String)`?  Then we don't need createSkipList().

Reply:
Thanks for that tip! I'll make the change.